presto-docs/src/main/sphinx/functions/array.rst

```diff
@@ -227,8 +227,8 @@ Array Functions
 
 .. function:: array_top_n(array(T), int) -> array(T)
 
-    Returns an array of the top n elements from a given ``array``, sorted according to its natural descending order.
-    If n is smaller than the size of the given ``array``, the returned list will be the same size as the input instead of n.::
+    Returns an array of the top ``n`` elements from a given ``array``, sorted according to its natural descending order.
+    If ``n`` is smaller than the size of the given ``array``, the returned list will be the same size as the input instead of ``n``. ::
```

Comment:
I believe this should have been - 

```suggestion
    If ``n`` is larger than the size of the given ``array``, the returned list will be the same size as the input instead of ``n``. ::
```

Reply:
Thanks @imjalpreet! Changed.